code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -911,6 +911,9 @@ proc/ui_describe_reagents(atom/A)
 	var/angry = FALSE
 	var/angry_timer = 15
 	var/list/convertible_reagents = list("blood","bloodc","bloody_mary","bloody_scary","hemolymph")
+	var/poked = FALSE // poked by a syringe, will try to heal using synthflesh
+	var/pokes = 0 // each poke reduces efficiency
+	var/poke_timer = 2
```

Comment:
Would appreciate some comments on how `poke_timer` works, I think I get it, but it's done in imprecise Per Process Tick units so it's hard to understand from the variable definition alone